bindings/src/main/java/io/pravega/storage/extendeds3/ExtendedS3StorageConfig.java

```diff
@@ -30,6 +30,7 @@
     public static final Property<String> BUCKET = Property.named("bucket", "");
     public static final Property<String> PREFIX = Property.named("prefix", "/");
     public static final Property<Boolean> USENONEMATCH = Property.named("useNoneMatch", false);
+    public static final Property<Integer> SMALL_OBJECT_THRESHOLD = Property.named("smallObjectThreshold", 1024 * 1024);
```

Comment:
What does `smallObjectThreshold` mean? Someone who has no idea what this means will not be able to use this properly.

Second you must always include the unit of measure in your property names. Is this bytes, KB, MB, ..., TB?

Please name this something more appropriate, such as `nativeConcatThresholdBytes` or `multiPartUploadConcatThresholdBytes` .

Reply:
renamed